test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Waiters.cs

```diff
@@ -44,6 +44,8 @@ public static void WaitForFileToBeReadable(string file, TimeSpan timeout)
 
         public static void WaitForProcessToStop(int processId, TimeSpan timeout, bool expectedToStop, string errorMessage)
         {
+            Console.WriteLine($"Waiting for process {processId} to stop...");
```

Comment:
Since we don't have and likely won't get a guideline about noise from tests, :ok: